helm/orca/config/alerts-mapping.yaml

```diff
@@ -185,6 +185,416 @@ prometheus:
         kind: node
         properties:
           name: instance
+    - name: KubeCronJobRunning
+      source_mapping:
+        origin: kubernetes
+        kind: cronjob
+        properties:
+          name: cronjob
+          namespace: namespace
+    - name: KubeJobCompletion
+      source_mapping:
+        origin: kubernetes
+        kind: job
+        properties:
+          name: job_name
+          namespace: namespace
+    - name: KubeJobFailed
+      source_mapping:
+        origin: kubernetes
+        kind: job
+        properties:
+          name: job_name
+          namespace: namespace
+    - name: KubeAPIDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeSchedulerDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeControllerManagerDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeCPUOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeMemOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeMemoryOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeCPUQuotaOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: namespace
+    - name: KubeMemoryQuotaOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: namespace
```

Comment:
Shouldn't we map these alerts to the cluster kind instead of the namespace kind?

Although the `kube_resourcequota` metric is namespace-scoped, [alert expressions](https://github.com/coreos/kube-prometheus/blob/a8b4985de4dc8582dcaf8fd43d6f88a0c08c463d/manifests/prometheus-rules.yaml#L1127-L1131) for these two alerts sum values for all metric instances (namespaces):
```yaml
- alert: KubeCPUQuotaOvercommit
    annotations:
    message: Cluster has overcommitted CPU resource requests for Namespaces.
    runbook_url: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubecpuquotaovercommit
    expr: |
    sum(kube_resourcequota{job="kube-state-metrics", type="hard", resource="cpu"})
        /
    sum(kube_node_status_allocatable_cpu_cores)
        > 1.5
```
The alert name (`for Namespaces`) is misleading though..